pyproject.toml

```diff
@@ -100,23 +94,39 @@ markers = ["unstable"]
 line-length = 99
 target-version = ["py38"]
 
-[tool.isort]
-profile = "black"
-
 # Linting tools configuration
-[tool.flake8]
-max-line-length = 99
-max-doc-length = 99
-max-complexity = 10
-exclude = [".git", "__pycache__", ".tox", "build", "dist", "*.egg_info", "venv"]
-select = ["E", "W", "F", "C", "N", "R", "D", "H"]
-# Ignore W503, E501 because using black creates errors with this
+[tool.ruff]
+target-version = "py38"
```

Comment:
Black is configured so and charm libs must be 3.8 compatible, but IMHO we can set 3.10 for pure jammy charms.

Reply:
Must charm libs be 3.8 compatible due to Juju 2.9 or something like that?

Reply:
So I think it's ok to keep the PG charm lib compatible with PgBouncer VM (which supports focal).

Reply:
We have to keep focal compatibility at least!